src/flows/components/Sidebar.tsx

```diff
@@ -182,7 +182,10 @@ const Sidebar: FC = () => {
               return true
             }
 
-            if (!/^(inputs|transform)$/.test(PIPE_DEFINITIONS[type]?.family)) {
+            if (
+              !/^(inputs|transform)$/.test(PIPE_DEFINITIONS[type]?.family) &&
+              flow.data.byID[id].type !== 'visualization'
```

Comment:
also should this be `type === 'visualization'`? Do you have a gif of this working?

Reply:
As commented below, it works, i guess tho we need to put in the .visual in there for it to fully function, i missed that it wasn't there when it converted to flux